src/k-provider/ovp/provider.js

```diff
@@ -40,7 +43,9 @@ export default class OVPProvider extends BaseProvider<ProviderMediaInfoObject> {
           ks = '{1:result:ks}';
           this._dataLoader.add(OVPSessionLoader, {partnerId: this.partnerId});
         }
-        this._dataLoader.add(OVPMediaEntryLoader, {entryId, ks});
+        const redirectFromEntryId =
```

Comment:
this code snippet and the one in the `getEntryListConfig` seem very much the same, better create a static method to resolve this and give back `redirectFromEntryId` value

Reply:
fixed